server/converters/cxgtool.py

```diff
@@ -6,26 +6,27 @@
 
     the.cxg                 TileDB Group
     |-- obs                 TileDB array containing cell (row) attributes, one attribute per
-    |                       dataframe columm, shape (n_obs,)
+    |                       dataframe column, shape (n_obs,)
     |-- var                 TileDB array containing gene (column) attributes, with one attribute per
     |                       dataframe column, shape (n_var,)
-    |-- X                   Main count matrix as a 2D TileDB array, single unnanmed numeric attribute
+    |-- X                   Main count matrix as a 2D TileDB array, single unnamed numeric attribute
     |-- emb                 TileDB group, storing optional embeddings (group may be empty)
     |   |-- <name1>         TileDB Array, single anon attribute, ND numeric array, shape (n_obs, N)
     |-- cxg_group_metadata  Empty array used only to stash metadata about the overall object.
+    |   |-- colors          CXG colors object as described in https://github.com/chanzuckerberg/cellxgene/issues/1307
```

Comment:
please inline the definition - the github issue is not sufficient documentation (ie, just cut & paste the 5 lines that describe the dict layout)

Reply:
Also, given that these are only category colors, and there is no clear way to determine that from the data structure described  in the github issue, I recommend you rename this to something like `cxg_category_colors`